cmake/Modules/OpmSatellites.cmake

```diff
@@ -297,9 +297,9 @@ macro(opm_add_test TestName)
       if (CURTEST_DRIVER)
         set(CURTEST_COMMAND ${CURTEST_DRIVER} ${CURTEST_DRIVER_ARGS} ${CURTEST_EXE_NAME} ${CURTEST_TEST_ARGS})
       else()
-        set(CURTEST_COMMAND "${PROJECT_BINARY_DIR}/bin/${CURTEST_EXE_NAME}")
+        set(CURTEST_COMMAND ${PROJECT_BINARY_DIR}/bin/${CURTEST_EXE_NAME})
```

Comment:
luckily no. a list is ; separated, not " " separated. you only need to worry in a string since that is " " separated. one of the reasons i like to keep lists as lists.